tests/Fixer/FunctionNotation/PhpdocToReturnTypeFixerTest.php

```diff
@@ -394,6 +394,20 @@ function bar() {}
             '<?php /** @return int */ fn(): int => 1;',
             '<?php /** @return int */ fn() => 1;',
         ];
+
+        yield '$this accessor' => [
```

Comment:
Hey @Zales0123 ,
I let myself to update your testcase - now, it executes same test on PHP 8+, but also verifies that there is no change when executing below PHP 8.

unfortunately, your change is causing valid PHP 7.4 code to crash after changes applied by Fixer on PHP 7.4 runtime